### ISSUE TITLE
CLA information for PR not updated

### ISSUE THREAD

**Issue by martinfalk** [inviwo/inviwo#136]
When creating a pull request, the license check for CLA is not updated automatically.

20min after the PR has been created:
![image](https://user-images.githubusercontent.com/9251300/40623653-44e84e30-62a8-11e8-938d-61b45ea045e4.png)

**Comment by martinfalk** [comment]
Could the reason be that CLA information is pulled only after a successful build?

**Comment by ResearchDaniel** [comment]
I have changed so that the webhooks to cla-assistant is sent when pushing as well-. Let's see if it improves the situation.

**Comment by ResearchDaniel** [comment]
Could be related to this issue: https://github.com/cla-assistant/cla-assistant/issues/77 ?
Says that it would be resolved by changing email in commit:
https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user

**Comment by r-englund** [comment]
The webhook have turned from a gren checkmark to a red triangle 
![image](https://user-images.githubusercontent.com/534670/40745004-956c2c96-6456-11e8-8c93-a57a2741fd6e.png)
It was green the other day

**Comment by ResearchDaniel** [comment]
We tried sending more events but they were unsupported. Changed back to what I think is supported so it should turn green again.

**Comment by r-englund** [comment]
Might have fixed it. Seems the content type was wrong (was set to the x-www-form-...). 
After caching it to json it seems to work for a new pull request ( #147 ) 
![image](https://user-images.githubusercontent.com/534670/40881373-49b6d960-66c5-11e8-9f30-6f9d150a55a7.png)

**Comment by martinfalk** [comment]
seems to work. Closing this issue for now.
